models/User.js

```diff
@@ -13,7 +13,7 @@ User.add({
 	photo: { type: Types.CloudinaryImage, collapse: true },
 	password: { type: Types.Password, initial: true, required: false },
 }, 'Permissions', {
-	isProtected: { type: Boolean, noedit: true, hidden: true },
+	isProtected: { type: Boolean, noedit: true },
```

Comment:
Make this visible to highlight difference between super admin and ordinary accounts (the isProtected permission).